solr/contrib/ltr/src/java/org/apache/solr/ltr/LTRRescorer.java

```diff
@@ -153,8 +149,8 @@ public TopDocs rescore(IndexSearcher searcher, TopDocs firstPassTopDocs,
     return hits;
   }
 
-  public void scoreFeatures(IndexSearcher indexSearcher,
-                            int topN, LTRScoringQuery.ModelWeight modelWeight, ScoreDoc[] hits, List<LeafReaderContext> leaves,
+  private void scoreFeatures(IndexSearcher indexSearcher,
+                            LTRScoringQuery.ModelWeight modelWeight, ScoreDoc[] hits, List<LeafReaderContext> leaves,
```

Comment:
This here could be considered backwards incompatible (signature and behaviour change to a public method) but if #473  first marked the public method deprecated (in a prior Solr release) then it could be considered a backwards compatible change.